### PR TITLE
Redesign feedback

### PR DESCRIPTION
**Pull request by ghislaineguerin**
- [x] It should be obvious that you are on the simplabs website (Logo)
- [x] Balance white-space
- [x] Fix titles services
- [x] Fix background images
- [x] Remove hover effect from the blog
- [x] Fix arrow alignment
- [x] Click target on cards
- [x] Add open source projects names
- [x] Fix blog posts padding
- [x] Fix padding on mobile
- [x] Clickable area
- [x] Fix scrolling issue with the jumbotron
- [x] Remove all caps for the author
- [x] Fix Typo on Ember card
- [x] Make Work button dark
- [x] Replace about us photo
- [x] Remove office photo
- [x] Fix line illustration on mobile
- [x] We don’t have images for the news
- [x] Fix blog preview
- [x] The map does not seem to point to the correct address
- [x] The address is missing line breaks
- [x] add client _"Cardstack"_ (see https://simplabs.com/clients/) (could add _"Shuddle"_, _"campus"_ and _"Bold Poker"_ if we need more to fill up a line with 4 items)
- [x] replace logos on start page: we should show trainline, Generali, Clark, Cardstack, Experteer
- [x] Make gray text black

### CODE REVIEW

**Comment by marcoow** [comment]
@ghislaineguerin: I can do this one.

> - [ ] Fix blog preview

**Comment by wadie** [comment]
I'm working on pointing the arrow to the correct address on the map. do you think this looks good ?

screenshot_2018-12-04 contact us

Even though our office is not exactly at that point, but it's on that street and in that position the name `Claude-Lorrain-Str.` appears clearly on the map.

**Comment by marcoow** [comment]
I'd point it at the correct point actually :)

**Comment by ghislaineguerin** [comment]
@wadie I agree with Marco, let’s point to the right spot. Thanks!

**Comment by ghislaineguerin** [comment]
@marcoow I've checked the list and added images for the news. 
image

**Comment by marcoow** [comment]
@ghislaineguerin that looks good but we actually wanted to change some of the news - see the suggestions in my reviews:

* https://github.com/simplabs/simplabs.github.io/pull/219#pullrequestreview-180205507
* https://github.com/simplabs/simplabs.github.io/pull/219#pullrequestreview-180157200

**Comment by ghislaineguerin** [comment]
@marcoow I've updated the news, please verify. thanks!

File: index.html [hunk 1]
```diff
@@ -146,27 +149,36 @@ <h2 class="display-4 font-weight-black text-lowercase mb-4">News</h2>
...
       <div class="col-sm-12 col-md-4">
+        {% capture news-3 %}
+          We are happy to launch Breethe, a Progressive Web App for accessing air quality data for locations around the world. We built Breethe with Glimmer.js so that it would be blazing fast and lightweight. It is server side rendered, works offline and even with JavaScript disabled!
+        {% endcapture %}
         {% include news_card.html 
-          title="CodeBot supports ES2017"
-          content="Our Slack Bot CodeBot, that takes any JavaScript code you send it, runs it and returns with the result and captured log output is now running Node 9 and thus supports all the latest and greatest of JavaScript."
+          title="Launching Breethe"
+          content=news-3
           url="http://codebot.chat/"
```
Comment (marcoow) [inline index.html]:
```suggestion
          url="https://breethe.app/"
```

**Review by marcoow — COMMENTED** [review]
LGTM 👍 1 news has a wrong link though

**Pull request merged**